starlite/types/composite_types.py

```diff
@@ -59,3 +59,4 @@
     Callable[..., ASGIApp], DefineMiddleware, Iterator[Tuple[ASGIApp, Dict[str, Any]]], Type[MiddlewareProtocol]
 ]
 MaybePartial = Union[T, partial]
+TypeEncodersMap = Dict[Any, Callable[[Any], Any]]
```

Comment:
Should the callables return any or bytes?

Reply:
`Any` is correct here. They only transform values into types that `msgspec` can understand, they don't serialize anything themselves.